.github/workflows/test.yaml

```diff
@@ -70,24 +70,26 @@ jobs:
             fail-fast: false
             matrix:
                 php-version: ['8.1', '8.3', '8.4']
+                symfony-version: ['6.4', '7.2']
                 minimum-stability: ['stable', 'dev']
-                include:
-                  - php-version: '8.1'
-                    dependency-version: 'lowest'
-                  - php-version: '8.3'
-                    dependency-version: 'highest'
-                  - php-version: '8.4'
-                    dependency-version: 'highest'
+                dependency-version: ['lowest', 'highest']
                 component: ${{ fromJson(needs.tests-php-components.outputs.components )}}
                 exclude:
                   - php-version: '8.1'
                     minimum-stability: 'dev'
                   - php-version: '8.3'
                     minimum-stability: 'dev'
+                  - php-version: '8.1'
+                    dependency-version: 'highest'
+                  - php-version: '8.3'
+                    dependency-version: 'lowest'
+                  - php-version: '8.4'
+                    dependency-version: 'lowest'
                   - component: Swup  # has no tests
                   - component: Turbo # has its own workflow (test-turbo.yml)
                   - component: Typed  # has no tests
-
+        env:
+            SYMFONY_REQUIRE: ${{ matrix.config.symfony-version }}
```

Comment:
be careful. Defining `SYMFONY_REQUIRE: 6.4` does not allow installing 6.4.1 or newer. It needs to be `6.4.*` as it is a constraint (`6.4` as constraints means `=6.4.0.0`)

Reply:
Indeed, thanks for noticing it!